src/dbt/kipptaf/models/illuminate/intermediate/int_illuminate__psat_unpivot.sql

```diff
@@ -5,6 +5,9 @@ select
     score,
 
     concat('psat10_', score_type) as score_type,
+    row_number() over (
+        partition by local_student_id, score_type order by score desc
+    ) as rn_highest,
```

Comment:
did you confirm the output looks right after this change? a row number at the same time as an unpivot can lead to unexpected results sometimes